src/cc65/symtab.c

```diff
@@ -700,14 +700,16 @@ SymEntry* AddEnumSym (const char* Name, const Type* Type, SymTable* Tab)
         } else {
             /* Define the struct size if the underlying type is given. */
             if (Type != 0) {
-                if (Type !=0 && Entry->V.E.Type != 0) {
+                if (Type != 0 && Entry->V.E.Type != 0) {
```

Comment:
The above two lines test `Type != 0` twice.

Reply:
Fixed.